lib/components/ms-input/MsAddressInput.vue

```diff
@@ -1,20 +1,27 @@
 <!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
 
 <template>
-  <ms-input
-    :label="label"
-    :placeholder="placeholder"
-    id="ms-address-input"
-    @change="onChange"
-    @ion-blur="onFocusLost"
-    v-model="address"
-    :debounce="timeBeforeQuery"
-  />
+  <div class="address-container">
+    <ms-input
+      :label="label"
+      :placeholder="placeholder"
+      id="ms-address-input"
+      @change="onChange"
+      @ion-blur="onFocusLost"
+      v-model="address"
+      :debounce="timeBeforeQuery"
+    />
+    <ms-address-dropdown
+      v-if="addressesFound.length > 0 && querying"
```

Comment:
The dropdown should not be shown while querying. The condition should only be `addressesFound.length > 0`